### DISCUSSION TITLE
Erro 404

### DISCUSSION THREAD

**Discussion by Jemerson07** [community/community#70385] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Fiz meu projeto e tentei hospedar com o github me encontrei com o erro 404 quando tento acessar o site, pesquisei e li diversas formas para solução do erro e ele continua.   encontrei erro na build with jekyll

**Comment by Dineth-H** [comment]
O erro 404 no seu site do GitHub Pages provavelmente é devido a um problema com o arquivo **__config.yml_**. Para corrigir isso, verifique o arquivo para se certificar de que está correto e em um formato YAML válido. Se o arquivo estiver correto, tente reconstruir seu site com o seguinte comando:

**jekyll build**

Se você ainda estiver recebendo o erro, tente limpar o cache do seu navegador. Se você ainda estiver tendo problemas, abra um problema no GitHub para obter ajuda.

Aqui estão algumas coisas específicas para verificar no arquivo **__config.yml_**:

- [ ] Certifique-se de que a variável **_url_** esteja configurada corretamente para o URL do seu site do GitHub Pages.
- [ ] Certifique-se de que a variável **_baseurl_** esteja configurada corretamente para o caminho do seu site no diretório raiz do seu repositório.
- [ ] Certifique-se de que a variável **_theme_** esteja configurada corretamente para o nome do tema que você deseja usar para seu site.
Espero que isso ajude!

**Comment by seyLu** [comment]
#### Hi  @Jemerson07,

I [deployed a copy of your site](https://seylu.github.io/Jemerson07.github.io/) and it works just fine.

From the logs, the issue was you were building and deploying to `docs` directory without ever having a `docs` directory -- hence the error.
![image](https://github.com/community/community/assets/98249191/778c5cf5-52b0-45ae-b933-254aab1a65d2)

Just change the build and deploy path to root `/` and you should be good to go.
![image](https://github.com/community/community/assets/98249191/5ef1b13d-f60f-463e-b104-cfc5fca2ebf9)
